v2/bigtable-changestreams-to-hbase/README.md

```diff
@@ -8,17 +8,33 @@ The [BigtableToHBase](src/main/java/com/google/cloud/teleport/v2/templates/Bigta
 * Bigtable table with change streams enabled
 * Hbase instance that is accessible from Dataflow
 
-#### Beware
+#### Beware - provision Hbase resources to handle Dataflow traffic
 
 * Dataflow when strongly provisioned can translate Bigtable input QPS directly to Hbase, the user should configure Hbase so that Hbase can handle that QPS without crashing.
+
 * Bigtable change streams can handle very large single mutations. Hbase write buffers should be adequately provisioned so as not to cause exceptions when writing in large mutations.
 
+#### Beware - out-of-order writes can occur in some scenarios
+
+The replicator writes to Hbase as a side effect in Dataflow. If Dataflow loses connection to a worker, another worker will be spun up and a change stream mutation could be duplicated. In the following scenario, a duplication could occur:
+  * Dataflow spins up worker A
+  * Worker A replicates a Put
+  * Dataflow loses connection to worker A before the Put is replicated and spins up worker B
+  * Dataflow assigns the Put to worker B, now 2 workers write the same Put
+
+The above scenario is not a problem most times. However, if a major compaction occurs and a cell had a masking Delete, then a delayed Put will be applied to a cell that should've been deleted. This scenario is theoretically possible but extremely unlikely.
+There are some strategies to deal with this scenario:
+  * Do not write in Deletes.
+  * Disable major compaction.
+  * Set Hbase `NEW_VERSION_BEHAVIOR = false`. A duplicated Put that arrived later will always be masked by a previous Delete.
+  * Set Hbase `KEEP_DELETED_CELLS = true` (see [here](https://hbase.apache.org/book.html#cf.keep.deleted)). Deletes continue to mask puts even after major compaction. With infinite max_versions and TTL, this is effectively the same as disabling major compaction.
+
 ### Notes on Bidirectional Replication
 
 This template can be configured to be used with the [Hbase-Bigtable replicator](https://github.com/googleapis/java-bigtable-hbase/blob/main/hbase-migration-tools/bigtable-hbase-replication/README.md) out of the box.
 To configure, enable `bidirectional replication` settings in the `Running Template` section below.
 
-#### Beware
+#### Beware - multi-master writes are not supported
```

Comment:
There is no way for you to stop multi-master setup. 

You should say, there is a probability of divergence if the same row is modified on the 2 databases simultaneously.